Info-Orbs/include/widgets/webDataElement.h

```diff
@@ -0,0 +1,21 @@
+#ifndef WEB_DATA_ELEMENT_H
+#define WEB_DATA_ELEMENT_H
+
+#include <ArduinoJson.h>
+#include <TFT_eSPI.h>
+
+class WebDataElement {
```

Comment:
It would probably make sense to move this class (and all things derived from it) into a folder `WebData` or something,
because this is an implementation detail of the `WebDataModel` and not a widget.

Reply:
I moved all data models to a `model` directory